WolvenKit.App/ViewModels/Tools/PropertiesViewModel.cs

```diff
@@ -347,19 +347,27 @@ public async Task LoadImageFromStream(Stream stream)
 
         public void LoadModel(RedBaseClass cls)
         {
-            var meshes = MakePreviewMesh(cls);
+            try
+            {
+                var meshes = MakePreviewMesh(cls);
 
-            IsMeshPreviewVisible = true;
-            SelectedIndex = 1;
+                IsMeshPreviewVisible = true;
+                SelectedIndex = 1;
 
-            var bounds = new SharpDX.BoundingBox();
-            foreach (var mesh in meshes)
+                var bounds = new SharpDX.BoundingBox();
+                foreach (var mesh in meshes)
+                {
+                    bounds = SharpDX.BoundingBox.Merge(bounds, mesh.BoundsWithTransform);
+                }
+
+                ModelGroupBounds = bounds;
+                ModelGroup.Reset(meshes);
+            }
+            catch
             {
-                bounds = SharpDX.BoundingBox.Merge(bounds, mesh.BoundsWithTransform);
+
```

Comment:
log the error?

Reply:
uhhh ... pretty sure that's not something i wrote, that's concerning